crates/matrix-sdk-ui/src/timeline/event_handler.rs

```diff
@@ -846,11 +866,7 @@ impl<'a, 'o> TimelineEventHandler<'a, 'o> {
     }
 
     /// Add a new event item in the timeline.
-    fn add(&mut self, should_add: bool, content: TimelineItemContent) {
-        if !should_add {
-            return;
-        }
```

Comment:
I seem to recall I had done the same change at some point and was requested changes, but that makes two of us agreeing with it now, so let's do it :muscle:

Reply:
\o/